### PR TITLE
Fix PiPiP storybook page with slim wrapper around PiPiP component

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Since upgrading storybook to version 6.5.7 in #1972, the PiPiP component storybook page does not load and times out.

I explored redefining the `primaryTile.children` ReactNode sub property to `primaryTile` direct ReactNode property and similarly `secondaryTile.children` ReactNode sub property to `secondaryTile` direct ReactNode property.
This fixes PiPiP storybook page. It seems like ReactNode/JSX.Element sub properties don't work in storybook version 6.5.7

# Why

https://skype.visualstudio.com/SPOOL/_workitems/edit/2918197

# How Tested

See generated chromatic link.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: change/@internal-storybook-25132f7a-8366-4fe3-9b0a-8dd72b96d04e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Updated code in PictureInPictureInPicture story.",
```
Comment (prprabhu-ms) [inline change/@internal-storybook-25132f7a-8366-4fe3-9b0a-8dd72b96d04e.json:3]: nit: The changelogs should be a single line that has enough context to understand what the change did. Imagine being the release train engineer and seeing this description with a 100 other changelog entries. Would you understand enough to know how to summarize the change for CHANGELOG.md

**Comment by prprabhu-ms** [comment]
I'm afraid I don't quite understand why the changed props fix the original issue. I also think that this PR changes the props for the worse. So, I'd like to understand why it is needed better, and document the reason in the code for posterity.

(I am still hopeful that we don't need to change the props, and that we can instead fix some bug in the story itself)

**Comment by mgamis-msft** [comment]
> I'm afraid I don't quite understand why the changed props fix the original issue. I also think that this PR changes the props for the worse. So, I'd like to understand why it is needed better, and document the reason in the code for posterity.
> 
> (I am still hopeful that we don't need to change the props, and that we can instead fix some bug in the story itself)

It seems Object properties with React.ReactNode/JSX.Element property do not work in storybook v6.5,7. Would  the prop change in [commit ](https://github.com/Azure/communication-ui-library/commit/dc4b5735d7bb1f866d99800db2907eeb3b73a32a) be better?

**Comment by prprabhu-ms** [comment]
> > I'm afraid I don't quite understand why the changed props fix the original issue. I also think that this PR changes the props for the worse. So, I'd like to understand why it is needed better, and document the reason in the code for posterity.
> > (I am still hopeful that we don't need to change the props, and that we can instead fix some bug in the story itself)
> 
> It seems Object properties with React.ReactNode/JSX.Element properties like `primaryTile` and `secondaryTile` of PiPiP do not work in storybook v6.5,7. Would the prop change in [commit ](https://github.com/Azure/communication-ui-library/commit/dc4b5735d7bb1f866d99800db2907eeb3b73a32a) be better?

Instead of modifying the component to fit storybook, can we create a slim wrapper component within the story (perhaps with the getTile() hack) and display that?

**Comment by mgamis-msft** [comment]
> > > I'm afraid I don't quite understand why the changed props fix the original issue. I also think that this PR changes the props for the worse. So, I'd like to understand why it is needed better, and document the reason in the code for posterity.
> > > (I am still hopeful that we don't need to change the props, and that we can instead fix some bug in the story itself)
> > 
> > 
> > It seems Object properties with React.ReactNode/JSX.Element properties like `primaryTile` and `secondaryTile` of PiPiP do not work in storybook v6.5,7. Would the prop change in [commit ](https://github.com/Azure/communication-ui-library/commit/dc4b5735d7bb1f866d99800db2907eeb3b73a32a) be better?
> 
> Instead of modifying the component to fit storybook, can we create a slim wrapper component within the story (perhaps with the getTile() hack) and display that?

Yep, that worked. No internal api changes needed. Thanks!

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**